spec/features/production/deployment_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "spec_helper"
+
+RSpec.describe "suspenders:production:deployment", type: :generator do
+  it "generates the configuration for a production deployment" do
+    rm "bin/deploy"
+
+    with_app { generate("suspenders:production:deployment") }
+
+    expect("bin/deploy").to exist_as_a_file
+    expect("bin/deploy").to be_executable
+    expect("README.md").to match_contents(%r{bin/deploy})
+  end
+
+  it "destroys the configuration for a production deployment" do
+    touch "bin/deploy"
+
+    with_app { destroy("suspenders:production:deployment") }
+
+    expect("bin/deploy").not_to exist_as_a_file
+    expect("README.md").not_to match_contents(%r{bin/deploy})
```

Comment:
Does this spec have any value if we don't write to the README first?

Reply:
I've no idea. In the least it's documentation, and at worst it makes sure we don't slip something about `bin/deploy` into the README unless it's through this generator.

Actually writing to the README first would be kinda tedious so I'm leaning toward not doing it.